frontend/app/components/Spots/SpotsList/index.tsx

```diff
@@ -132,27 +133,20 @@ function SpotsList() {
 
   return (
     <div className={'w-full relative'}>
-
       <div className={'flex mx-auto p-2 px-4 bg-white rounded-lg shadow-sm mb-2 w-full z-50'}>
         <SpotsListHeader
           showDeleteButton={selectedSpots.length > 0}
           onDelete={batchDelete}
+          selectedCount={selectedSpots.length}
```

Comment:
if we adding count we can as well remove `showDeleteButton` and re-declare it inside the component

Reply:
I've updated the code to avoid the redundancy you called out.